### PR TITLE
Admin grammar fixes and improvements

### PR DESCRIPTION
**Pull request by pgaskin**
Admin grammar fixes and improvements.

### CODE REVIEW

File: options/locale/locale_en-US.ini
Comment (pgaskin) [inline options/locale/locale_en-US.ini:1245]: The above ones are checkboxes on the user edit page, so no need to say `This account`. This is why I changed these lines.

File: options/locale/locale_en-US.ini
Comment (cez81) [inline options/locale/locale_en-US.ini]: transferred

Comment (pgaskin) [reply options/locale/locale_en-US.ini]: Thanks

**Comment by cez81** [comment]
LGTM

File: options/locale/locale_en-US.ini
Comment (bkcsoft) [inline options/locale/locale_en-US.ini]: This should be `Disabled if you are using the built-in SSH server.` no? As in it won't do anything...

File: options/locale/locale_en-US.ini
Comment (bkcsoft) [inline options/locale/locale_en-US.ini]: `All public keys controlled by Gitea [...]`

File: options/locale/locale_en-US.ini
Comment (bkcsoft) [inline options/locale/locale_en-US.ini]: Is this true? Is it not only if you're an Owner of an Org?

File: options/locale/locale_en-US.ini
Comment (bkcsoft) [inline options/locale/locale_en-US.ini]: Maybe `Only use a read-only account` ?

**Review by bkcsoft — COMMENTED** [review]
I know it's just a grammar-check, but we might as well touch some of them up while we're at it 😉

**Comment by andreynering** [comment]
LGTM

**Comment by lafriks** [comment]
Please do not merge this before 1.2.0 is released as this will break already translated strings

**Review by bkcsoft — CHANGES_REQUESTED** [review]
@lafriks You can do empty "request changes" to "stall" PRs ;)

**Comment by lafriks** [comment]
@geek1011 rebase

**Comment by lunny** [comment]
Any update? Or we should close this one?

**Comment by pgaskin** [comment]
Sorry, I have been quite busy for the last few weeks. I will update it soon.

**Comment by pgaskin** [comment]
@lunny @bkcsoft 

Also, sorry for the delay, I have been really busy recently.

**Comment by lafriks** [comment]
@bkcsoft need your approval

**Comment by lunny** [comment]
@lafriks CI failed

**Comment by lunny** [comment]
need @bkcsoft approval

**Review by bkcsoft — APPROVED** [review]

**Comment by lafriks** [comment]
Make LG-TM work again

**Pull request merged**
